chrome/content/coniks/context.js

```diff
@@ -0,0 +1,41 @@
+this.EXPORTED_SYMBOLS = ["AccountContext", "ConiksContext"];
+
+var { interfaces: Ci, utils: Cu, classes: Cc } = Components;
+
+Cu.import("chrome://otr/content/helpers.js");
+
+class ConiksContext {
+  constructor(opts) {
+    opts = opts || {};
```

Comment:
I'm not sure I want to use it, it seems to give me no control on what field should be assigned/included in the `ContextConiks`.